### PR TITLE
Update CXX to 1.12.0 release.

### PR DESCRIPTION
**Pull request by markmandel**

### CODE REVIEW

**Comment by mcos** [comment]
This looks great. Is there a chance you could update the other languages to the same version (we're dependent on the php, python and go images, specifically)? Thanks either way.

**Comment by markmandel** [comment]
@mcos if you want to submit PRs, and get the approved, I can get the images up on Dockerhub.

**Comment by mcos** [comment]
cool. I can make some as soon as i have some time. cheers.

**Comment by markmandel** [comment]
Gentle bump, can I get an approval on this PR?

File: 1.12.0/cxx/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Copyright 2017 gRPC authors.
```
Comment (nathanielmanistaatgoogle) [inline 1.12.0/cxx/Dockerfile]: Copied from one of last year's sources?

**Review by nathanielmanistaatgoogle — CHANGES_REQUESTED** [review]

File: 1.12.0/cxx/Dockerfile (hunk 1 above)
Comment (markmandel) [reply 1.12.0/cxx/Dockerfile]: Fixed!

**Review by nathanielmanistaatgoogle — APPROVED** [review]

**Comment by markmandel** [comment]
Coming back to this - any blocking issues on getting this merged?

**Review by nicolasnoble — APPROVED** [review]

**Comment by markmandel** [comment]
Thank you - build is currently in queue:
https://hub.docker.com/r/grpc/cxx/builds/

**Pull request merged**